lib/exq/redis/job_stat.ex

```diff
@@ -177,4 +198,49 @@ defmodule Exq.Redis.JobStat do
         val
     end
   end
+
+  defp supervisor_worker_name(namespace, process_info) do
+    JobQueue.full_key(namespace, "#{process_info.hostname}:elixir")
+  end
+
+  defp serialize_processing_payload(nil) do
+    %{}
+  end
+
+  defp serialize_processing_payload(job) do
+    %{
+      queue: job.queue,
+      class: job.class,
+      args: job.args,
+      jid: job.jid,
+      created_at: job.enqueued_at,
+      enqueued_at: job.enqueued_at
+    }
+  end
+
+  def status_process_commands(namespace, node_id, started_at, master_pid, queues, work_table, poll_timeout) do
+    name = redis_worker_name(namespace, node_id)
+    [
+      ["SADD", JobQueue.full_key(namespace, "processes"), name],
+      ["HSET", name, "quiet", "false"],
+      ["HSET", name, "info", Poison.encode!(%{ hostname: node_id, started_at: started_at, pid: "#{:erlang.pid_to_list(master_pid)}", concurrency: concurrency_count(queues, work_table), queues: queues})],
```

Comment:
I ran this branch locally and couldn't get it to work. Sidekiq tries to fetch info, but since this function is never called by anything, it's empty always.